### PR TITLE
tweaks

### PR DESCRIPTION
**Pull request by mixmix**
I added a couple of lines about running the sbot

I also took the liberty to tweak a couple of things that I thought could be clearer: 
- I never clone to `/tmp` and not being a linux guru, I know it's a 'special place'. I changed the instructions so that people were not confused / distracted about git-ssb being particular to some special linux path.
- I changed the direction of install instructions for Patchwork to the most up to date (and maintained) location.
- I changed the order of the git servers to reflect the support level (mine is almost definitely down).

Other notes : serving images from the git-ssb servers is a good way to get no image... I recommend copying an image into this repo so people get a great first impression)

### CODE REVIEW

**Comment by mixmix** [comment]
Further, I tried to simplify the install / setup section to make it easier and quicker to follow : 
- added numbers 1, 2, 3 to make it clear how much was involved
- trimmed the invite section (as that's covered well in the patchwork install)

File: README.md [hunk 1]
```diff
@@ -52,53 +55,54 @@ protocol runs in the background and propagates new content to everyone involved
...
-Let's use `npm` to install the `git-ssb` command:
+Let's use `npm` to install `git-ssb` globally so we can use it in the command-line:
 
 ```
 $ npm install --global git-ssb
 ```
 
-You can run `git-ssb` (or `git ssb`) from the command line now and get a feel
-for the sorts of things it can do
+You can now run `git-ssb` (or `git ssb`) from the command line now and get a feel for the sorts of things it can do.
```
Comment (hackergrrl) [inline README.md]: Double "now" -- could you prune one?

File: README.md [hunk 2]
```diff
@@ -157,16 +160,14 @@ request:
...
-
 $ git clone ssb://%RPKzL382v2fAia5HuDNHD5kkFdlP7bGvXQApSXqOBwc=.sha256 git-ssb-intro
 
 $ cd git-ssb-intro
 ```
 
 ### Fork
 
-Let's look at the help for `git-ssb fork`:
+Let's look check out the help `git-ssb fork --help`:
```
Comment (hackergrrl) [inline README.md]: Did you mean to pick one of "Let's look at" or "Let's check out" here? :wink:

**Review by hackergrrl — CHANGES_REQUESTED** [review]

**Comment by hackergrrl** [comment]
Wow, thank you mix! These changes are great, and I really appreciate the detailed itemized list you wrote up of the changes and your thinking behind each. I made two small comments, but otherwise everything you've done is a solid improvement to what's already there. :star2:

**Comment by mixmix** [comment]
nice catches on my sloppy typos! thanks for review and clear feedback.
pushed some changes

**Review by hackergrrl — APPROVED** [review]

**Comment by hackergrrl** [comment]
Perfect! I'll push these changes to git-ssb as well. Thanks again!

**Pull request merged**